C/plugins/north/OMF/plugin.cpp

```diff
@@ -897,7 +897,7 @@ uint32_t plugin_send(const PLUGIN_HANDLE handle,
 	// Check if the endpoint is PI Web API and if the PI Web API server is available
 	if (!IsPIWebAPIConnected(connInfo, version))
 	{
-		Logger::getLogger()->warn("PI Web API server %s is not available. Unable to send data to PI", connInfo->hostAndPort.c_str());
+		// Error already reported by IsPIWebAPIConnected
```

Comment:
Not quite true. If _IsPIWebAPIConnected_ determines that the server is not available, it will retry the next time around but no more often than once per minute. If this method is called before the minute has elapsed, it returns false with no message at all. This is probably fine because the first loss of connection is definitely logged. If we write a message in every call to _plugin_send_, there will probably be too many messages.

Reply:
Yes, and probably what we want as there is nothing to gain by repeatedly logging the fact the PIWebAPI is not available